backend/Squirrel.Core/Squirrel.Core.Common/Enums/EngineEnum.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Squirrel.Core.Common.Enums
+{
+    public enum EngineEnum
```

Comment:
Why is it called EngineEnum? Isn't this just DatabaseEngine?

Reply:
> Why is it called EngineEnum? Isn't this just DatabaseEngine?

I called it as it was specified in the database schema